app/src/main/java/nl/tudelft/b_b_w/model/block/BlockFactory.java

```diff
@@ -45,7 +45,7 @@ public static final Block createBlock(BlockData data) throws HashException {
     @Deprecated
     public static final Block getBlock(String type, String newOwner, int sequenceIndex, String hash,
```

Comment:
will this deprecated method be in the master ? Or is this just a temporary solution.